scripts/platform-init/platform-init.py

```diff
@@ -0,0 +1,202 @@
+from base64 import b64decode, b64encode
+import json
+import logging
+from typing import cast
+from kubernetes import client as k8s_client, config as k8s_config
+from dataclasses import dataclass, field
+import os
+from cryptography.hazmat.primitives.asymmetric import rsa
+from cryptography.hazmat.primitives import serialization
+
+
+@dataclass
+class Config:
+    k8s_namespace: str
+    renku_fullname: str
+    secret_service_public_key: str | None
+    secret_service_private_key: str | None = field(repr=False)
+
+    @classmethod
+    def from_env(cls):
+        config_map = json.loads(os.environ.get("PLATFORM_INIT_CONFIG", "{}"))
```

Comment:
See the other comment in the values file. It is nicer to use YAML. Otherwise we have YAML for most of the values file and then a huge section with json.

And when we switch to yaml people can still use json if they want to.